### PR TITLE
DON'T MERGE: Use Valkyrie's new Single Valued Support

### PR DESCRIPTION
**Pull request by tpendragon**
This PR is mostly just for the real Cho devs to look at.

References samvera-labs/valkyrie#470
Depends on samvera-labs/valkyrie#485

### CODE REVIEW

File: app/cho/data_dictionary/fields_for_object.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ module DataDictionary::FieldsForObject
       Rails.application.load_seed if DataDictionary::Field.all.empty? && Rails.env.test?
 
       DataDictionary::Field.all.each do |field|
+        next if schema.keys.include?(field.label.parameterize.underscore.to_sym)
```
Comment (awead) [inline app/cho/data_dictionary/fields_for_object.rb]: @tpendragon is this so we're not adding an attribute that's already defined?

Comment (tpendragon) [reply app/cho/data_dictionary/fields_for_object.rb]: Yeah, I was just having a hard time spinning up the app without it. Not necessary.

Comment (awead) [reply app/cho/data_dictionary/fields_for_object.rb]: @tpendragon if this makes things easier, then by all means, leave it in! Perhaps a brief comment that explains what it's easing...

Comment (tpendragon) [reply app/cho/data_dictionary/fields_for_object.rb]: Sure, just tossed a comment above it.

**Comment by awead** [comment]
Note: squash and merge.

**Comment by tpendragon** [comment]
Note: I'd probably wait to merge this until there's a released version.

**Comment by awead** [comment]
Replaced with #635

**Pull request closed without merging**